bash_it.sh

```diff
@@ -1,117 +1,129 @@
 #!/usr/bin/env bash
-# Initialize Bash It
-BASH_IT_LOG_PREFIX="core: main: "
 
 # Only set $BASH_IT if it's not already set
-if [ -z "$BASH_IT" ]; then
+if [[ -z "$BASH_IT" ]]; then
 	# Setting $BASH to maintain backwards compatibility
 	export BASH_IT=$BASH
 	BASH="$(bash -c 'echo $BASH')"
 	export BASH
 	BASH_IT_OLD_BASH_SETUP=true
 fi
 
-# Load composure first, so we support function metadata
+# Load composure first, to support function metadata, then create our custom attributes.
 # shellcheck disable=SC1090
 source "${BASH_IT}"/vendor/github.com/erichs/composure/composure.sh
+cite _about _param _example _group _author _version
+cite about-alias about-completion about-plugin
 
-# We need to load logging module first as well in order to be able to log
+# Next, load our logging library so we can give useful feedback to the user
 # shellcheck source=./lib/log.bash
 source "${BASH_IT}/lib/log.bash"
```

Comment:
add prefix and log that we load the file (using the helper function specified by my other comment)

Reply:
@NoahGorny Same thing here; the logging functions haven't been loaded yet - no logging can be done yet.

Reply:
I see, sorry, can you add a simple comment about it, so confused folks like me be less confused?